<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>465</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>345</width>
    <height>465</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1" colspan="4">
    <widget class="QLineEdit" name="squareEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="squareLabel">
     <property name="text">
      <string>Площадь:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="6">
    <widget class="QTableWidget" name="pointTable">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Точка</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>X</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Y</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="yLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="6">
    <widget class="QPushButton" name="removePointBtn">
     <property name="text">
      <string>Удалить точку</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="6">
    <widget class="QPushButton" name="addPointBtn">
     <property name="text">
      <string>Добавить точку</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="perimeterLabel">
     <property name="text">
      <string>Периметр:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="4">
    <widget class="QLineEdit" name="perimeterEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="xLabel">
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="xEdit"/>
   </item>
   <item row="6" column="2">
    <widget class="QLineEdit" name="yEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
